frontend/js/src/explore/art-creator/components/IconTray.tsx

```diff
@@ -83,6 +86,29 @@ function IconTray(props: IconTrayProps) {
             </span>
           )}
         </div>
+        {browserHasClipboardAPI && (
+          <button
+            type="button"
+            className="btn btn-icon btn-info"
+            onClick={onClickCopyAlt}
+            style={{
+              marginLeft: "auto",
+              background: "transparent",
+              color: "darkgray",
+            }}
+          >
+            <text
```

Comment:
Why this text element? It's an SVG tag that does not belong in this HTML.
Maybe you want a `<span>` instead ?